e2e/data/smoke/rook_api_external.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: v1
```

Comment:
I was replacing them at runtime, but I was recommended to hard code the port so that its always exposed on the same node. I can open a ticket and we can discuss this further.